skins/laika/src/components/pages/donation_confirmation/DonationSummary.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+	<div class="has-background-bright columns">
+		<div class="column is-half">
+			<div class="donation-summary">
+				<span class="payment-summary" v-html="getSummary()"></span>
+				<span class="payment-email" v-html="getEmail()"></span>
+				<span class="payment-notice" v-html="getPaymentNotice()"></span>
+				<BankData v-if="isBankDataVisible" :bank-transfer-code="confirmationData.donation.bankTransferCode"></BankData>
+			</div>
+		</div>
+		<div class="column is-half donation-links">
+			<a href="javascript:window.print()">{{ $t( 'donation_confirmation_print_confirmation' ) }}</a>
```

Comment:
this probably doesn't work? :thinking: 
afaik you can't have the `window` object directly in the template.

Reply:
Seems to work just fine, I guess because Vue does not care what is placed in a normal href tag and browsers just happen to know what to do with this "weird URL".